File: docs/examples/hello.py
"""hello.py - the example from https://aikaryashala.com/system_setup/04_install_py/

Run it with uv. There is no environment to create and nothing to activate:

    uv run hello.py
"""

import sys


def main() -> None:
    print(f"Hello from Python {sys.version.split()[0]}")


if __name__ == "__main__":
    main()

File: docs/examples/Hello.java
/* Hello.java - the example from
 * https://aikaryashala.com/system_setup/05_install_java/
 *
 * Compile and run:
 *   javac Hello.java
 *   java Hello
 *   java Hello Ubuntu
 *
 * Or run the source directly, without producing a .class file:
 *   java Hello.java
 */

public class Hello {
    public static void main(String[] args) {
        String who = args.length > 0 ? args[0] : "world";
        System.out.println("Hello, " + who + "!");
        System.out.println("Running on Java " + System.getProperty("java.version"));
    }
}
